src/01-Learning-JS/04-conditionals.md

```diff
@@ -175,7 +176,9 @@ if () {
 ```
 
 ```js
-// Your code here
+if (person1age == 28){
```

Comment:
Consider if `person1age` was assigned the value of `31`, it would still evaluate as `true`. Instead, consider

```suggestion
// use the less than operator instead (<)
if (person1age < 30) {
```